.github/workflows/fpga.yml

```diff
@@ -249,7 +249,11 @@ jobs:
                        package(caliptra-builder) |
                        package(caliptra-cfi-derive) |
                        package(caliptra-file-header-fix) |
-                       package(compliance-test))'
+                       package(compliance-test) |
+                       package(caliptra-drivers) |
+                       package(caliptra-runtime) |
+                       package(caliptra-test) |
```

Comment:
I think this could be reduced to `fips_test_suite`.

E.g. ` binary_id(caliptra-test::fips_test_suite)'`